src/lib/ng-package/entry-point/write-package.transform.ts

```diff
@@ -128,6 +74,70 @@ export const writePackageTransform = (options: NgPackagrOptions) =>
     return graph;
   });
 
+async function copyAssets(
+  graph: BuildGraph,
+  entryPointNode: EntryPointNode,
+  ngPackageNode: PackageNode,
+): Promise<void> {
+  const ngPackage = ngPackageNode.data;
+
+  const pathsIgnored: string[] = [
+    '.gitkeep',
+    '**/.DS_Store',
+    '**/Thumbs.db',
+    '**/node_modules/**',
+    `${ngPackage.dest}/**`,
+  ];
+
+  const assets: AssetEntry[] = [];
+
+  for (const item of ngPackage.assets.concat('LICENSE', '**/README.md')) {
+    let asset: AssetEntry;
+    if (typeof item == 'object') {
+      asset = item;
+      asset.glob = path.join(ngPackage.src, asset.input, asset.glob);
+    } else {
+      const glob = path.join(ngPackage.src, item);
```

Comment:
This is actually not a glob, hence better rename to something more meaningful 

```suggestion
      const assetPath = path.join(ngPackage.src, item);
```

Reply:
Hi, I really have no idea what else it can be, could you give me some examples?

Reply:
In this case `item` can be a glob, directory or file path, hence naming it `glob` doesn't seem appropriate. Therefore I recommand `assetPath`.

Reply:
Oh but when I saw a variable named as `xxPath`, I would think it as a pure path, and also file paths and directories are globs too, so maybe we can consider another name? or add a comment?